bin/alluxio

```diff
@@ -247,6 +185,7 @@ function copyDir {
 }
 
 PARAMETER=""
+ALLUXIO_SHELL_JAVA_OPTS=""
 
 case ${COMMAND} in
```

Comment:
for readability sake wrap this switch command in a main function that gets invoked from the bottom of the shell script (https://google.github.io/styleguide/shell.xml#main)